### ISSUE TITLE
 limited space of relationtable in identify

### ISSUE THREAD

**Issue by mtabdar** [roemhildtg/cmv-widgets#33]
i am using relation tables in identify popup for several layers of my app, and it is GREAT! (thanks for the widget)
just in some cases i prefer to have more space for the contents of tables, is there a way to enable horizontal scrolling? or is there a way other than limiting the number of columns?
from another point of view why identify popup is not resize-able?

**Comment by mtabdar** [comment]
I found that by changing width=100% to width==500px in factory.js identify becomes scrollable. I think i can live with that.

**Comment by roemhildtg** [comment]
Awesome glad you figured it out. Would you be willing to post the fix? For others it might be helpful

**Comment by mtabdar** [comment]
Yes. Of course. I would be honoured.though it is somehow too simple. 
In factory.js in line 67 which is
 style: 'width:100%;height:280px;'
I replaced width:100% with some pixel value. 
style: 'width:500px;height:280px;'
As simple as that. It has though some shortcomings:
The property tab also gets this size so it is a bit misplaced from the center of the window. 
And it is a fixed size for all identify windows whether you have a small tabe or a large one
And the vertical scroll is at the end of table so you have to move horizontal scroll to right end then scroll down -mouse wheel does not need that.
